### PR TITLE
add optional delay to forever mode

### PR DESCRIPTION
**Pull request by shekenahglory**

### CODE REVIEW

**Comment by wilsonianb** [comment]
Can you update the README?

**Comment by shekenahglory** [comment]
yessir

**Comment by shekenahglory** [comment]
updated readme

File: index.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ commander
   .option('-l, --logsql',
           'Log all sequelize queries and ddl')
   .option('-m, --message',
-          'Send message for each crawl stored to db (needs -s) to sqs queue');
+          'Send message for each crawl stored to db (needs -s) to sqs queue')
+  .option('-d, --delay <delay>',
+          'delay between crawls with forever mode, in seconds');
```
Comment (wilsonianb) [inline index.js]: It looks like we should add `parseFloat` or `parseInt` as a third argument to `option`
https://www.npmjs.com/package/commander

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**